server/src/main/java/org/candlepin/resource/HypervisorResource.java

```diff
@@ -102,11 +83,14 @@
     private ConsumerType hypervisorType;
     private JobManager jobManager;
 
+    @Inject
```

Comment:
@Inject should not be necessary, principalProvider is injected via constructor.

Reply:
Removed `@Inject` annotation.
Initially, I tried with the injected value. But Junit tests were getting failed. Later added this as a constructor parameter. It remained because rspecs also failed. And now it's working after removing.